Source/Scene/GroundPrimitive.js

```diff
@@ -599,6 +601,8 @@ define([
             boundingVolumes = groundPrimitive._boundingVolumes;
         } else if (frameState.mode !== SceneMode.SCENE3D && defined(groundPrimitive._boundingVolumes2D)) {
             boundingVolumes = groundPrimitive._boundingVolumes2D;
+        } else {
+            throw new RuntimeError('Failed to compute ground primitive bounding volumes');
```

Comment:
This should never be hit. The only reason `_boundingVolumes2D` won't be defined is when the scene was constructed with `scene3DOnly : true`. If this is hit, there is a bug elsewhere since the scene mode shouldn't be able to change.

Reply:
So should this be a developer error?

Reply:
I would remove the error and the second condition to make it just an `else`.